enginecore/enginecore/cli/status.py

```diff
@@ -57,7 +58,11 @@ def status_table_format(assets, stdscr=False):
         stdscr (optional): default window return by initscr(),
                            status_table_format uses print if omitted
     """
-
+    # Catches empty model and doesnt throw error
+    if assets is None:
+        print("Table is empty")
+        sys.exit(0)
```

Comment:
Following the previous Author's conventions, use `return` instead of `sys.exit(0)` (unless there's a valid reason to use it).